pkg/logs/input/docker/parser.go

```diff
@@ -33,15 +33,31 @@ type parser struct {
 	logParser.Parser
 }
 
-// Parse extracts the date and the status from the raw docker message
-// see https://godoc.org/github.com/moby/moby/client#Client.ContainerLogs
+// Parse calls parse to extract the message body, status, timestamp
+// can put them in the Message
 func (p *parser) Parse(msg []byte) (*message.Message, error) {
+	body, status, timestamp, err := parse(msg)
+	parsedMsg := message.NewMessage(body, nil, status)
+	parsedMsg.Timestamp = timestamp
+	return parsedMsg, err
+}
+
+// Unwrap removes the message header of docker logs
+// return log body and the timestamp
+func (p *parser) Unwrap(line []byte) ([]byte, string, error) {
+	body, _, timestamp, err := parse(line)
+	return body, timestamp, err
+}
+
+// parse extracts the date and the status from the raw docker message
+// see https://godoc.org/github.com/moby/moby/client#Client.ContainerLogs
+func parse(msg []byte) ([]byte, string, string, error) {
```

Comment:
nits: it's not clear that this function returns a message, a severity, a timestamp and an error, can we make this clearer please ?